dist/rust/riot-rust-linker-hook

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+[ -z "$BINDIR" ] && {
+    echo "$0: BINDIR not defined!" 1&>2
+    exit 1
+}
+
+_EXTRA_LIBS=${BINDIR}/_extra_libs
+mkdir -p ${_EXTRA_LIBS}
+
+while true; do
+    [ -z "$1" ] && exit 0
+    [ -f "$1" ] && {
+        if [[ "$1" == *.o ]]; then
```

Comment:
This line breaks on dash (and thus probably on posix shells).
Any other nice portable way of comparing the suffix?